drivers/storage/portworx/util/util.go

```diff
@@ -438,6 +440,13 @@ func KubeletPath(cluster *corev1.StorageCluster) string {
 	if IsPKS(cluster) {
 		return "/var/vcap/data/kubelet"
 	}
+
+	for _, env := range cluster.Spec.Env {
```

Comment:
can we move this before IsPKS check, it should be the same if uses choose to overwrite it in case of failures, I think the variations should also exist on PKS

Reply:
you are right, let me add that change